src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -144,11 +145,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String messageContent = request.getParameter("message");
 
     if(type != null){
-      if(!conversation.getType().equals(type)){
+      if(type.equals("make private")){
```

Comment:
If we switch to using a toggle button (e.g. `<input type="checkbox" value="isPrivate">`), we can use `isPrivate` as a form attribute all over, instead of having to send "type" and converting it.

Reply:
So the users will see a value named "isPrivate" and we showed it checked or not depending on the state of the conversation?

Reply:
Yes, and it works the other way too. Clicking the checkbox will toggle the type. It's up to you of course, I was just offering an alternative experience.